circle.yml

```diff
@@ -1,3 +1,7 @@
+machine:
+  environment:
+    SAUCE_LOG: $CIRCLE_ARTIFACTS/sauce_log.txt
```

Comment:
This part not mandatory anymore with the latest `--wait` implementation. It is still nice to collect Sauce Connect's logs, but an intermediate variable might be overkill if used in only one place.